packages/jobs/lib/activities.ts

```diff
@@ -466,14 +478,21 @@ export async function runPostConnectionScript(args: PostConnectionScriptArgs): P
 
     const context: Context = Context.current();
 
-    const syncRun = new syncRunService({
+    const syncRun = new SyncRunService({
         bigQueryClient,
         integrationService,
         recordsService,
         slackService,
         writeToDb: true,
         nangoConnection,
-        syncName: name,
+        syncConfig: {
+            sync_name: name,
+            file_location,
+            models: [],
+            track_deletes: false,
+            type: 'sync',
+            version: '0' // TODO: pass the correct version
```

Comment:
Why not just pass the version from the `external-post-connection` script, since we already have it from the query https://github.com/NangoHQ/nango/blob/master/packages/server/lib/hooks/connection/external-post-connection.ts#L21

Reply:
changed 👌🏻